packages/main/src/MenuItem.ts

```diff
@@ -60,6 +63,16 @@ type MenuItemAccessibilityAttributes = Pick<AccessibilityAttributes, "ariaKeySho
 	styles: [ListItem.styles, menuItemCss],
 	dependencies: [...ListItem.dependencies, ResponsivePopover, List, BusyIndicator, Icon],
 })
+
+/**
+ * Fired when an item is being selected.
+ * @since 2.5.0
+ * @public
+ */
+@event("item-selection", {
```

Comment:
Event names must finish with a verb in present tense.
Why is the word `item` used everywhere? `item-selection` and `ItemSelectionMode.ts`/`itemSelectionMode` ? Why not just `select` and `selectionMode`? Is there anything else that can be selected, other than items?

Reply:
Done